adapters/repos/db/lsmkv/bucket_snapshot_test.go

```diff
@@ -160,15 +161,14 @@ func TestSnapshot_ListFiles(t *testing.T) {
 			err := b.Put([]byte(fmt.Sprint(i)), []byte(fmt.Sprint(i)))
 			require.Nil(t, err)
 		}
+		b.FlushMemtable(ctx)
```

Comment:
Maybe a comment here indicating why the memtable is being flushed

Reply:
It makes sense!
Done